gradle.properties

```diff
@@ -25,6 +25,8 @@ spring_platform_version=Brussels-SR7
 
 ## Override Spring Platform IO Versions
 
+flyway.version=4.2.0
```

Comment:
Is this property automagically used by gradle/nebula?

Reply:
It's used by the Spring dependency management plugin to swap in the version controlled by their boms. See [here](https://docs.spring.io/spring-boot/docs/2.0.0.RELEASE/gradle-plugin/reference/html//#managing-dependencies-customizing)